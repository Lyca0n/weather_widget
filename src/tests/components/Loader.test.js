import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../modules/common/Loader';

test('should correctly render LoadingPage', () => {
  const wrapper = shallow(<Loader />);
  expect(wrapper).toMatchSnapshot();
});
