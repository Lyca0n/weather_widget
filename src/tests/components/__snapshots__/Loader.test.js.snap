// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LoadingPage 1`] = `
<div
  className="loader-container"
>
  <Loader
    color="#00BFFF"
    height="2vh"
    type="Puff"
    width="2vw"
  />
</div>
`;
